services/spar/src/Spar/Scim/User.hs

```diff
@@ -600,20 +600,13 @@ synthesizeStoredUser tid usr = do
         when (managedBy /= ManagedByScim) $ do
           Brig.setBrigUserManagedBy uid ManagedByScim
 
-  (richInfo, accStatus, accessTimes, baseuri) <- lift $ readState
-  SAML.Time (toUTCTimeMillis -> now) <- lift $ SAML.getNow
+  (richInfoAssocList, accStatus, accessTimes, baseuri) <- lift readState
+  SAML.Time (toUTCTimeMillis -> now) <- lift SAML.getNow
   let (createdAt, lastUpdatedAt) = fromMaybe (now, now) accessTimes
-  storedUser <- synthesizeStoredUser' tid usr (synthesizeRichInfo richInfo) accStatus createdAt lastUpdatedAt baseuri
+  storedUser <- synthesizeStoredUser' tid usr (RichInfo.assocListToRichInfo richInfoAssocList) accStatus createdAt lastUpdatedAt baseuri
   lift $ writeState (BrigTypes.userId usr) accessTimes (BrigTypes.userManagedBy usr) storedUser
   pure storedUser
 
--- TODO: This function belongs to Wire.API.User.RichInfo and can be unit tested
```

Comment:
You said you would add a unit test once moving it to its rightful place; do we still want to do that?